packages/code-du-travail-frontend/pages/convention-collective/[slug].js

```diff
@@ -38,15 +45,61 @@ class ConventionCollective extends React.Component {
           image={ogImage}
         />
         <Answer
-          title={shortTitle}
+          breadcrumbs={[
+            {
+              label: "Recherche",
+              slug: getRouteBySource(SOURCES.CCN)
+            }
+          ]}
+          date={
+            convention.date_publi &&
+            format(parseISO(convention.date_publi), "dd/MM/yyyy", {
+              locale: frLocale
+            })
+          }
+          dateLabel="Entrée en vigueur le"
           emptyMessage="Cette convention collective n'a pas été trouvée"
-          wide
+          relatedItems={[
+            {
+              slug: "convention-collective",
+              source: SOURCES.SHEET_SP,
+              title: "Convention collective"
+            },
+            {
+              slug: "comment-consulter-un-accord-dentreprise",
+              source: SOURCES.SHEET_SP,
+              title: "Comment consulter un accord d'entreprise ?"
+            },
+            {
+              slug: "#hierarchie",
+              source: SOURCES.LABOUR_LAW,
+              title:
+                "Droit du travail: Existe-t-il une hiérarchie entre les textes ?"
+            }
+          ]}
+          source={{
+            name: "Légifrance",
+            url: convention.url
+          }}
+          subtitle={
+            <SmallText>
+              {convention.title} (IDCC {formatIdcc(convention.num)})
+            </SmallText>
+          }
+          suptitle="CONVENTION COLLECTIVE"
+          title={shortTitle}
         >
-          <Convention convention={convention} />
+          <Convention {...convention} />
```

Comment:
vu qu'il n y' a que 3 props, je pense qu'on peut les passer de manière explicite

Reply:
As you prefer !